data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1843 @@
+			`	The slippery snow certainly doesn't help her hooves, but it does make the crate slippery as well. It's coaxed out of the ground, bit by bit. You help her as best as you can and eventually get it on top of the snow field. Iplora searches through some tools in her bag and takes out what looks like a crowbar. She starts opening it, a dim light creeping out, but an energy blast coming from the the top of the hills hits the ground near the hole. It misses you by just about a meter.`
+			`	She yells for you to get down, pulling out her weapon and firing in the direction the shot came from.`
+			apply
+				"resistance evidence" ++
+			choice
+				`	(Take cover.)`
+				`	(Pull out your gun and help her.)`
+					goto help
+			`	You get inside the hole, hoping you'll have enough protection from the blasts. You can still hear Iplora move around, and you occasionally peek your head over to see her shooting and taking cover, running and dodging incoming blasts. The shots stop and you see her come over, saying that it's safe to get out now.`
+			`	"Fled, our attacker has," she claims. From the looks of it she wasn't injured. You two get back to the crate.`
+				goto opencrate
+			label help
+			`	You start shooting the hills above, trying to aim around where Iplora's own shots go to. A shot comes your way, and you jump to the side. Iplora takes cover inside the hole, firing a few more shots from there, then leaves up the hill. You follow her lead, approaching from another direction.`
+			`	The shots alternate targets, some firing at Iplora, then some at you, then back at her. The blizzard is bad enough that you can't even see the attacker, but it seems that goes both ways, as you and Iplora manage to avoid getting hit without too much trouble.`
+			`	You've barely gotten past the last cave you visited when the shots stop altogether. Iplora still takes aim for a bit, but ultimately lowers her weapon, coming toward you. "Fled, our attacker has," she claims. You two walk back to the crate.`
+			label opencrate
+			`	When it's finally open, you see it contains a Heliarch finisher torpedo. You two go around the fields, and find that many other mounds are identical crates. She uses a communication device and calls for Heliarch support. Several Heliarch Neutralizers arrive some time after the blizzard subsides, and you're directed to one of them to dry your clothes and make sure you're not frostbitten.`
+			`	Once you leave, you see several Heliarch agents using proper equipment to dig out the crates and opening them dozens at a time, revealing several more torpedoes. A group has also made their way up the hills looking for any sign of whoever attacked you. Iplora speaks with some of the Heliarchs there, then tells you that they will handle everything from here. One of the ships takes both of you to where your own ship landed and leaves as you prepare to take off.`
```

Comment:
So this is the glow - why do they glow?

Reply:
They have a glow in the outfit images.
Just thought it'd be neat imagery, but don't mind removing the glow bit.